src/__tests__/ProcessTest.js

```diff
@@ -76,7 +101,10 @@ const kill = function(reason) {
     test.spies().kill(reason);
 };
 
-const write = chunk => test.spies().stdin(chunk);
+const restoreAndSet = function(toRestore, toSet, functionName, stubFunction) {
```

Comment:
hier kannsde dir glaub ich durch beforeEach bzw afterEach das leben wesentlich leichter machen, wenn ich das richtig sehe (die werden jeweils vor und nach jedem it gecalled)